src/chat/components/InputToolbar.tsx

```diff
@@ -83,7 +86,14 @@ const InputToolbar: React.FC<IInputToolbar> = ({
       {!!text && (
         <PressableIcon
           iconStyle={flattenedIconStyle}
-          onPress={() => onSend?.({ text: text }, true)}
+          onPress={() => {
+            if (isFilterBadWords) {
+              const replaceBadWords = filterBadWords(text, 'en');
```

Comment:
onSend?.({ text: filterBadWords(text, 'en') }, true);

Reply:
I removed it. It's no need